app/src/main/java/com/example/mvp/ui/signup/SignUpPresenter.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.mvp.ui.signup
+import com.example.mvp.data.model.user
+
+class SignUpPresenter : SignUpContract.Presenter {
+
+    private var mView : SignUpContract.View? = null
+    val listUser = mutableListOf<user>()
+
+    fun setView(view: SignUpContract.View){
+        mView = view
+    }
+
+
+    override fun handleSignUp(userName: String, passWord: String) {
+        if(userName != " " && passWord != " "){
```

Comment:
```suggestion
        if(userName.notBlankOrEmpty() && passWord.notBlankOrEmpty()){
```
Tên hàm có thể không đúng